vectors_benchmark/src/binaries/1m_stats.rs

```diff
@@ -41,7 +41,7 @@ fn label_set(batch_id: usize) -> Vec<String> {
 
 fn add_batch(
     writer: &mut Index,
-    batch_id: String,
+    _batch_id: String,
```

Comment:
Please remove this parameter 🐱

Reply:
Done at [349c9e9](https://github.com/nuclia/nucliadb/pull/428/commits/349c9e90ef4540f50001078123025ec50dee56e4)